internal/golden_file_test/bin.js

```diff
@@ -5,13 +5,13 @@ function main(args) {
   const [mode, golden_no_debug, golden_debug, actual] = args;
   const debug = process.env['COMPILATION_MODE'] === 'dbg';
   const golden = debug ? golden_debug : golden_no_debug;
-  const actualContents = fs.readFileSync(require.resolve(actual), 'utf-8').replace(/\r\n/g, '\n');
-  const goldenContents = fs.readFileSync(require.resolve(golden), 'utf-8').replace(/\r\n/g, '\n');
+  const actualContents = fs.readFileSync(actual, 'utf-8').replace(/\r\n/g, '\n');
```

Comment:
will we be able to explain to users where they might need to make a similar change?

Reply:
backed this out